Code/MinimalLib/cffi_test.c

```diff
@@ -489,6 +490,78 @@ void test_modifications() {
   printf("--------------------------\n");
 }
 
+float **_get_coord_array(char *mpkl, size_t mpkl_size) {
```

Comment:
wow, writing this shows serious dedication to testing. Thanks! :-)

Reply:
Writing C code is like driving a classic car: no assisted steering and traction control but, oh, how charming.